src/ets/scala/io/iohk/ethereum/ets/blockchain/BlockchainSuite.scala

```diff
@@ -23,7 +23,7 @@ class BlockchainSuite extends FreeSpec with Matchers with Logger {
           (name, scenario) <- group.scenarios
           if options.isScenarioIncluded(name)
         } {
-          name in new ScenarioSetup(scenario) {
+          name in new ScenarioSetup(options, scenario) {
```

Comment:
Could we create the VM here in `BlockchainSuite` instead of `ScenarioSetup` (no need to pass options to setup, just the VM)? Then we could properly close/destroy it in the `afterAll()` method.

The reason for doing so is, when running the tests more than once:
- the VM doesn't pick up logging/config changes without restarting SBT
- an ugly connection error appears (even though the tests pass)

Reply:
Sounds good, I'll do that